Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean

```diff
@@ -100,23 +89,45 @@ lemma inv_rnDeriv [SigmaFinite μ] [SigmaFinite ν] (hμν : μ ≪ ν) (hνμ :
     (by filter_upwards [hνμ.ae_le (rnDeriv_pos hμν)] with x hx using hx.ne')
     (rnDeriv_ne_top _ _)]
 
+lemma set_lintegral_rnDeriv_eq_withDensity {s : Set α} (hs : MeasurableSet s) :
+    ∫⁻ x in s, μ.rnDeriv ν x ∂ν = ν.withDensity (μ.rnDeriv ν) s :=
+  (withDensity_apply _ hs).symm
+
+lemma set_lintegral_rnDeriv_le {s : Set α} (hs : MeasurableSet s) :
+    ∫⁻ x in s, μ.rnDeriv ν x ∂ν ≤ μ s := by
+  rw [set_lintegral_rnDeriv_eq_withDensity hs]
+  exact withDensity_rnDeriv_le _ _ _ hs
+
 lemma set_lintegral_rnDeriv [HaveLebesgueDecomposition μ ν] (hμν : μ ≪ ν) {s : Set α}
```

Comment:
Is it really necessary in these two lemmas that `s` should be measurable?

Reply:
I removed the `MeasurableSet s` assumption in the two lemmas with inequalities.

Reply:
I am not sure I can remove it for the equalities, can I?

Reply:
I only meant for the inequality lemmas, just as you did.